torch/_dynamo/variables/builder.py

```diff
@@ -1250,6 +1250,7 @@ def wrap_to_fake_tensor_and_record(
         )
         if is_tensor and not (static_shapes and source.is_nn_module()):
             tx.output.tracked_fakes.append(TrackedFake(fake_e, source, constraint_dims))
+        tx.output.tensor_id_to_fake_clone[id(e)] = fake_e.clone()
```

Comment:
Given how you're using this, there is no reason to clone the entire fake tensor. Just extract and save the sizes and strides.

Reply:
Sure, we can do that.